parser/v2/types.go

```diff
@@ -415,29 +417,28 @@ func (e Element) Write(w io.Writer, indent int) error {
 	if err := writeIndent(w, indent, "<"+e.Name); err != nil {
 		return err
 	}
-	var previousWasMultiline bool
```

Comment:
we no longer care about multi line because the parser for conditionals requires a newline after the `{` which would populate IndentAttrs property through parsing logic.

Reply:
I think that putting the state into the element type makes a lot of sense. It makes the code neater too.